library/std/src/thread/mod.rs

```diff
@@ -738,6 +738,8 @@ pub fn panicking() -> bool {
 ///
 /// This function is blocking, and should not be used in `async` functions.
 ///
+/// The behavior when given a zero duration is platform-specific.
+///
 /// # Platform-specific behavior
```

Comment:
Isn't this implied by the 'platform-specific' heading just below?

Reply:
Also, this function is deprecated, so you may want to document `std::thread::sleep` instead.

Reply:
We could still update deprecated functions but usually I recall we remove examples from deprecated function to encourage them to use the newer ones.

Reply:
Apologies, I intended to update the non-deprecated sleep() function instead of this one!

Reply:
It would be better if you could stash or rebase it to reduce unnecessary commits.